rotkehlchen/tests/unit/globaldb/test_upgrades.py

```diff
@@ -483,7 +490,18 @@ def test_upgrade_v5_v6(globaldb):
         assert gen_cache_unique_key_content == unique_cache_content
         assert gen_cache_content_before == gen_cache_content_after + gen_cache_unique_key_content
 
-        assert cursor.execute('SELECT * FROM multiasset_mappings ORDER BY collection_id, asset').fetchall() == old_multiasset_mappings  # noqa: E501
+        assert cursor.execute('SELECT * FROM multiasset_mappings ORDER BY collection_id, asset LIMIT 232').fetchall() == old_multiasset_mappings  # noqa: E501
+        assert cursor.execute(
+            'SELECT COUNT(*) FROM multiasset_mappings',
+        ).fetchone() == old_multiasset_mappings_count
+        assert cursor.execute(
```

Comment:
Wouldn't a better and more complete check be to process the identifiers, extract the addresses and make sure they are all checksummed?